extensions/rest-raml/README.md

```diff
@@ -0,0 +1,19 @@
+# @loopback/rest-raml
+
+## Installation
+
+Install LoopbackRestRamlComponent using `npm`;
+
+```sh
+$ [npm install | yarn add] @loopback/rest-raml
+```
+
+## Basic Use
+
+Configure and load LoopbackRestRamlComponent in the application constructor as
+shown below.
+
+```ts
+import {LoopbackRestRamlComponent} from '@loopback/rest-raml';
+this.component(LoopbackRestRamlComponent);
+```
```

Comment:
Can you please add some instructions now how to obtain the RAML spec/document for application's API? Do we have any plans to add a CLI command for that too?

Reply:
> Can you please add some instructions on how to obtain the RAML spec/document for application's API? Do we have any plans to add a CLI command for that too?

I think this comment haven't been addressed yet.